server/src/main/java/io/crate/types/DataTypes.java

```diff
@@ -76,6 +77,8 @@
     public static final TimestampType TIMESTAMPZ = TimestampType.INSTANCE_WITH_TZ;
     public static final TimestampType TIMESTAMP = TimestampType.INSTANCE_WITHOUT_TZ;
 
+    public static final DateType DATE = DateType.INSTANCE;
```

Comment:
Could you also add the type to `STORAGE_UNSUPPORTED` for now and add documentation to https://github.com/crate/crate/blob/1b7b2bfdd4e950068be7f11e69c48585369d0767/docs/general/ddl/data-types.rst#L36-L48 ?

Reply:
Done

Reply:
@mfussenegger, I added DATE type to the DataTypes.STORAGE_UNSUPPORTED and updated DataTypeTesting ALL_TYPES_EXCEPT_ARRAYS -  `ALL_TYPES_EXCEPT_ARRAYS.removeIf(DataTypes.STORAGE_UNSUPPORTED :: contains).`

Basically, on my machine `LuceneQueryBuilderIntegrationTest testNullOperators` was failing with INTERVAL - I noticed, that it's also UNSUPPORTED so commented it out. `removeIf `covers INTERVAL exclusion but I decided keep `ALL_TYPES_EXCEPT_ARRAYS.add(DataTypes.INTERVAL)` just in case and commented it out, so that it's not silently excluded.

INTERVAL is being selected constantly by randomness.getRandom - I don't know is it my local issue or some extra sees is needed but anyway it's not failing anymore.